### PR TITLE
Add detail of responses for the error status codes to Numbers API

### PR DESCRIPTION
**Pull request by lornajane**
# New 

* Describe error responses in Numbers API

# Checklist

- [x] version number incremented (in the `info` section of the spec)

### CODE REVIEW

File: definitions/numbers.yml [hunk 1]
```diff
@@ -58,16 +58,10 @@ paths:
...
-                    description: A code relating to this error
-                    example: "401"
-                  error-code-label:
-                    type: string
-                    description: Words describing the error that occurred
-                    example: authentication failed
+                $ref: "#/components/schemas/unauthorized"
+            text/xml:
+              schema:
+                $ref: "#/components/schemas/unauthorized"
```
Comment (mheap) [inline definitions/numbers.yml:64]: These seem to be missing `xml.name` which controls the name of the wrapper. In the API it's `response` but in the rendered version it's `hash`

**Comment by lornajane** [comment]
This was a good catch but it was a bit of a can of worms since most of the responses were missing the XML root tag info! Added for all endpoints, rebased the branch and bumped the version number. This should be in good shape now!

File: definitions/numbers.yml (hunk 1 above)
Comment (lornajane) [reply definitions/numbers.yml]: Good spot. This was already missing from the other endpoints too which I don't think helped but I've added them all now!

**Comment by mheap** [comment]
🙌 Thanks for seeing this through

**Pull request merged**